### DISCUSSION TITLE
Can we get support for PDF files ?

### DISCUSSION THREAD

**Discussion by MrTHP** [n4ze3m/page-assist#88] [Ideas]
basically the idea is sometimes i have pdf documents that i need to translate from english to french. google or deepl or whatever site are limited in the size of documents to 15mb. in local with this page assist tool would be nice if we can add upload pdf or other document formats and ask any llm to please translate this document in french please. i know it would be long depending on the size but i dont care i just want it to work :P we could also talk we our documents and ask questions about anything in it. later on doing this with a youtube link would be cool you take a joe rogan podcast link then you ask a question about something you heard in the video but forgot about it. llm would search on the fly and give us the responses back.

**Comment by n4ze3m** [comment]
This is a nice idea. Currently, Page Assist supports PDF parsing, but I don't think it's good for translation, only for RAG. Anyway, I will try it. ;)
